### PR TITLE
Docs: Subnets README - Added steps to build and test on Subnets

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
Added the following to the README file.

- Instructions to build Subnets
- Instructions to run tests

**Updated files:**
- README.md
- NFT_USE_CASE.md

### CODE REVIEW

File: NFT_USE_CASE.md [hunk 1]
```diff
@@ -267,4 +267,4 @@ principal now owns the NFT (`ALT_USER_ADDR`). You can check this by clicking on
 
 
 That is the conclusion of this demo! If you have any issues with this demo, reach out on the Stacks Discord or leave an issue in the 
-stacks-hyperchains repository.
\ No newline at end of file
+stacks-hyperchains repository.
```
Comment (gregorycoppola) [inline NFT_USE_CASE.md:270]: subnets?

File: README.md [hunk 2]
```diff
@@ -41,12 +41,444 @@ In a fully - trusted model:
...
+> **_NOTE:_**
+>
+> Subnets were referred to as Hyperchains. While we are in the process of updating the content, you might still see subnets being referred to as Hyperchains.
+
+## Build with Clarinet
+
+Make sure you have `clarinet` installed locally, and that it is at version 0.33.0 or above.
+If you do not have clarinet, you can find installation instructions [here](https://github.com/hirosystems/clarinet).
+
+Let's create a new clarinet project. This will create a new directory with a Clarinet project initialized.
```
Comment (gregorycoppola) [inline README.md:61]: you have `clarinet` and the `Clarinet`

**Review by gregorycoppola — COMMENTED** [review]
Thanks @LakshmiLavanyaKasturi !

I have a higher level question..

I thought we were going to _link_ to the other `.md` files (like `DEPLOYING.md`), instead of copying the into the root-level README.

What this a conscious choice to copy the md texts into the root level?

File: README.md [hunk 3]
```diff
@@ -41,12 +41,444 @@ In a fully - trusted model:
...
+
+Let us copy contract files and scripts over from the `stacks-hyperchains` repository into the `nft-use-case` directory. 
+If you don't already have the stacks-subnets repository, you can [clone it](https://github.com/hirosystems/stacks-subnets).
+Here's the command to clone the stacks-hyperchains repository:
+```
+git clone https://github.com/hirosystems/stacks-hyperchains.git
+```
+Set the environment variable `HYPERCHAIN_PATH` to the location of the stacks-hyperchains repository on your computer. 
+```
+export HYPERCHAIN_PATH=<YOUR_PATH_HERE>
```
[the remark this answers is not in the document]
Comment (pavitthrap) [reply README.md:74]: You might want to actually see how the files you pulled from (NFT_USE_CASE.md, DEPLOYING.md, LOCAL_TESTING.md) were modified. Not sure how you want to go about integrating those changes.

**Comment by LakshmiLavanyaKasturi** [comment]
> Thanks @LakshmiLavanyaKasturi !
> 
> I have a higher level question..
> 
> I thought we were going to _link_ to the other `.md` files (like `DEPLOYING.md`), instead of copying the into the root-level README.
> 
> What this a conscious choice to copy the md texts into the root level?

@gregorycoppola: It was by mistake. I have added the links now. Also, I have changed hyperchains-> subnets throughout the repo. I understand there is another PR: https://github.com/hirosystems/stacks-subnets/pull/162 with changes, but just to avoid merge conflicts, we can probably include everything in one PR. Suggestions?

**Comment by LakshmiLavanyaKasturi** [comment]
Paused work on this item to focus on Stacks 2.1 updates. Will resume soon.

**Review by obycode — COMMENTED** [review]
I rebased these changes on top of the latest master and resolved the conflicts from the "hyperchains" -> "subnets" renaming. @LakshmiLavanyaKasturi please review to make sure my rebase didn't cause any problems.

File: README.md [hunk 4]
```diff
@@ -41,14 +41,49 @@ In a fully - trusted model:
...
+
+## Run Tests
+
+In your terminal, you can run tests by navigating to the `testnet/stacks-node/` directory and running the following command:
+
+`testnet/stacks-node$ cargo test`
+
+If you want to ignore some tests, you can use the following command:
+
+`testnet/stacks-node$ cargo test -- --ignored --num-threads=1`
```
Comment (obycode) [inline README.md:79]:
```suggestion
`cargo test -- --ignored --num-threads=1`
```

**Comment by LakshmiLavanyaKasturi** [comment]
LGTM. Thanks, @obycode for these changes. I will update the license info and we should be good to go.

**Comment by LakshmiLavanyaKasturi** [comment]
> In your commit to add the license info, you also modified the NFT_USE_CASE.md and scripts/ files. I don't think those should have changed. Was that intentional?
> 
> For example, `SUBNET_URL` --> `SUBNETS_URL`. I think the previous `SUBNET_URL` is better, since it refers to one subnet.

Thanks for catching these. I've updated the URLs. Can you review it now? Thanks!

File: core-contracts/contracts/multi-miner.clar [hunk 5]
```diff
@@ -81,4 +81,4 @@
          ;; check that we have enough signatures
          (try! (check-miners (append (get signers signer-principals) tx-sender)))
          ;; execute the block commit
-         (as-contract (contract-call? .subnet commit-block (get block block-data) (get target-tip block-data) (get withdrawal-root block-data)))))
+         (as-contract (contract-call? .subnets commit-block (get block block-data) (get target-tip block-data) (get withdrawal-root block-data)))))
```
Comment (obycode) [inline core-contracts/contracts/multi-miner.clar:84]: This one should also be reverted. The contract is just `subnet` now.

**Review by obycode — APPROVED** [review]
Thanks!

**Pull request merged**